macros/edr/metadata_collection/get_columns_in_project.sql

```diff
@@ -0,0 +1,20 @@
+{% macro get_columns_in_project() %}
+    {% set configured_schemas = elementary.get_project_configured_schemas() %}
+    {{ elementary.get_columns_by_configured_schemas(configured_schemas) }}
+{% endmacro %}
+
+{% macro get_project_configured_schemas() %}
```

Comment:
Is it still used? Can it be replaced by your `macro`?

Reply:
It is still in use, the schema changes tests use it. 
can't be replaced, has a different logic - tasks from the graph resource type `test`, where name start with `schema_changes` and package not `elementary`

Reply:
Yes but the point there was to bring only the columns we must and not all project columns.
Now we bring all the columns anyway, so that logic is outdated.